### PR TITLE
remove duplicate value in case clause

### PR DESCRIPTION
**Pull request by dustinxie**
some small issues when I pick the PR to r1.6 branch

### CODE REVIEW

File: action/const.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ var (
 // LoadErrorDescription loads corresponding description related to the error
 func LoadErrorDescription(err error) string {
 	switch errors.Cause(err) {
-	case ErrOversizedData, ErrTxPoolOverflow, ErrInvalidSender, ErrNonceTooHigh, ErrInsufficientFunds, ErrIntrinsicGas, ErrChainID, ErrNotFound, ErrVotee, ErrAddress, ErrExistedInPool, ErrReplaceUnderpriced, ErrNonceTooLow, ErrUnderpriced, ErrNonceTooHigh, ErrAddress, ErrNegativeValue:
+	case ErrOversizedData, ErrTxPoolOverflow, ErrInvalidSender, ErrNonceTooHigh, ErrInsufficientFunds, ErrIntrinsicGas, ErrChainID, ErrNotFound, ErrVotee, ErrAddress, ErrExistedInPool, ErrReplaceUnderpriced, ErrNonceTooLow, ErrUnderpriced, ErrNegativeValue:
```
Comment (dustinxie) [inline action/const.go:60]: 2 duplicate values

File: action/action.go [hunk 2]
```diff
@@ -119,7 +119,10 @@ func ClassifyActions(actions []SealedEnvelope) ([]*Transfer, []*Execution) {
 }
 
 func calculateIntrinsicGas(baseIntrinsicGas uint64, payloadGas uint64, payloadSize uint64) (uint64, error) {
-	if payloadGas == 0 || (math.MaxUint64-baseIntrinsicGas)/payloadGas < payloadSize {
+	if payloadGas == 0 {
+		return 0, ErrUnderpriced
```
Comment (dustinxie) [inline action/action.go:123]: 0 < minimum gas price, `ErrUnderpriced` is more proper

File: action/protocol/execution/evm/evm.go [hunk 3]
```diff
@@ -453,7 +453,7 @@ func evmErrToErrStatusCode(evmErr error, g genesis.Blockchain, height uint64) (e
 // intrinsicGas returns the intrinsic gas of an execution
 func intrinsicGas(data []byte) (uint64, error) {
 	dataSize := uint64(len(data))
-	if action.ExecutionDataGas == 0 || (math.MaxInt64-action.ExecutionBaseIntrinsicGas)/action.ExecutionDataGas < dataSize {
+	if (math.MaxInt64-action.ExecutionBaseIntrinsicGas)/action.ExecutionDataGas < dataSize {
```
Comment (dustinxie) [inline action/protocol/execution/evm/evm.go:456]: `action.ExecutionDataGas = 100`

File: actpool/actpool.go [hunk 4]
```diff
@@ -218,8 +218,8 @@ func (ap *actPool) Add(ctx context.Context, act action.SealedEnvelope) error {
 	if act.GasPrice().Cmp(ap.cfg.MinGasPrice()) < 0 {
 		actpoolMtc.WithLabelValues("gasPriceLower").Inc()
 		log.L().Info("action rejected due to low gas price",
-			zap.String("act hash", hex.EncodeToString(hash[:])),
-			zap.String("gas price", act.GasPrice().String()))
+			zap.String("actionHash", hex.EncodeToString(hash[:])),
```
Comment (dustinxie) [inline actpool/actpool.go:221]: use "actionHash" to be consistent with other places

File: action/action.go (hunk 2 above)
Comment (Liuhaai) [reply action/action.go:123]: Why is the later condition removed

File: action/const.go (hunk 1 above)
Comment (Liuhaai) [reply action/const.go:60]: Good finding

**Review by Liuhaai — APPROVED** [review]

**Review by raullenchai — APPROVED** [review]

**Pull request merged**